README.md

```diff
@@ -1,5 +1,16 @@
 # SnapIT HacktoberFest
 
+## Installation
+
+Install Node Version:
+`nvm install v14.17.3`
+Install Yarn
+`https://classic.yarnpkg.com/en/docs/install/`
+Install Packages
+`yarn install`
+Run Application
+`yarn dev`
+
```

Comment:
⛏️ Add an empty line between each instruction. Currently they're displaying in one line.